CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -65,42 +65,44 @@ class HomeViewController: UIViewController {
         view.addSubview(tabCollectionView)
 
         setUpConstraints()
+        setUserAndTabPage()
```

Comment:
Just a question: do we want networking to be done in viewWillAppear? Asking cause when you just call once in viewDidLoad, if there are any changes and you come back to this screen, the changes won't actually appear.